geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionStats.java

```diff
@@ -1253,13 +1256,18 @@ protected void incSerialQueueSize(int messages) {
   }
 
   protected void incSerialQueueBytes(int amount) {
+    serialQueueBytes.addAndGet(amount);
     this.stats.incInt(serialQueueBytesId, amount);
   }
 
   public int getSerialQueueBytes() {
     return this.stats.getInt(serialQueueBytesId);
   }
 
+  public int getInternalSerialQueueBytes() {
```

Comment:
When converting to `AtomicLong` or `LongAdder` please update these methods to longs.